integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/specs/EscapeSpecification.groovy

```diff
@@ -41,9 +40,11 @@ class EscapeSpecification extends GebReportingSpec {
     when: 'I go to the tasklist page'
 
     elite2api.stubUncategorised()
```

Comment:
I simplified the sorting to not need the reverse() but this caused some problems with the tests, so i forced
the dates to be different in the stubs